### ISSUE TITLE
Example is not working.

### ISSUE THREAD

**Issue by TheArmagan** [SharkFinPro/razer-chroma-nodejs#3]
```
Error: connect ECONNREFUSED 127.0.0.1:62165
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 62165
}
```

i am getting this error.

**Comment by SharkFinPro** [comment]
You must have the syntapse app installed.

**Comment by SharkFinPro** [comment]
my bad, this was actually part of a larger issue that has been fixed as of version 3.1.0
